hardware/areca.py

```diff
@@ -26,51 +26,52 @@
 
 
 def split_units(lis):
-    'If the value have unit, remove it from the value and return the unit name'
+    # If the value have unit, remove it from the value
+    # and return the unit name
```

Comment:
maybe make an actual docstring out of this comment?

Reply:
yeah......